src/side-nav/side-nav-group.styles.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+`;
+
+export const Divider = styled.hr`
+    margin: 1rem 1rem 0 1rem;
```

Comment:
to specify the required css for styling. I believe we're trying to move away from sgds css